src/templates/react/Content/src/App.fs

```diff
@@ -0,0 +1,152 @@
+namespace FableElmishReactTemplate
+
+open Elmish
+open Elmish.Browser.Navigation
+open Elmish.Browser.UrlParser
+open Fable.Core
+open Fable.Core.JsInterop
+open Fable.Import
+open Fable.Import.Browser
+open FableElmishReactTemplate.Common
+
+module App =
+
+  [<Emit("require('../sass/main.sass');")>]
+  let requireStyle () = ()
+
+  requireStyle()
```

Comment:
I was thinking react need `require` keyword but this seems to work too :)
And it's cleaner.